core/com.b2international.snowowl.datastore/src/com/b2international/snowowl/core/validation/ValidateRequestBuilder.java

```diff
@@ -28,20 +29,28 @@
 public final class ValidateRequestBuilder 
 		extends BaseRequestBuilder<ValidateRequestBuilder, BranchContext, ValidationResult>
 		implements RevisionIndexRequestBuilder<ValidationResult> {
-
+	
 	ValidateRequestBuilder() {}
 	
 	private Collection<String> ruleIds;
 	
+	private Map<String, Object> filterOptions;
+	
 	public ValidateRequestBuilder setRuleIds(Collection<String> ruleIds) {
 		this.ruleIds = ruleIds;
 		return getSelf();
 	}
 	
+	public ValidateRequestBuilder setFilterOptions(Map<String, Object> filterOptions) {
```

Comment:
`ruleParameters` would be a better name for this request parameter, rename it pls.